spec/card_generator_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rspec'
+require 'pry'
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+require './lib/card_generator'
+
+RSpec.describe CardGenerator do
+
+  xit 'exists' do
```

Comment:
@cjsim89

Question about this - whenever I initially make a class and corresponding 'exists' test, it seems like I should do it without arguments so I'm _only_ testing that it exists, without anything else going on. But then when I move on to other tests where I start needing to add arguments to the class, that initial test starts failing because of wrong number of arguments. 

Is it better to go back and change the 'exist' test to pass as I need to add arguments, or skip it so I don't constantly get notification that the test is failing?